src/app/src/components/VideoPlayer.js

```diff
@@ -94,7 +94,7 @@ const StyledVideo = styled(Video)`
 `;
 
 const VideoPlayer = ({ src }) => {
-    const [isPlaying, setIsPlaying] = useState(true);
+    const [isPlaying, setIsPlaying] = useState(false);
```

Comment:
Sorry if there was some confusion. The autoplay on initial open is good, but when the user interacts with the scrubber then the video should pause. We probably should just roll back this commit.

Reply:
OK thanks for clarifying! What a mess I made. The real problem turned out to be that the video controls would go out of sync to the video, see commit msg b4bafb6, which I've fixed. I had accidentally broken autoplay in this PR, which you've caught, which is also fixed in b4bafb6